packages/slate-editor/src/extensions/placeholders/elements/StoryBookmarkPlaceholderElement.tsx

```diff
@@ -0,0 +1,104 @@
+import type { StoryRef } from '@prezly/sdk';
+import React from 'react';
+import { useSlateStatic } from 'slate-react';
+
+import { SearchInput } from '#components';
+import { PlaceholderStory } from '#icons';
+import { useFunction } from '#lib';
+
+import { EventsEditor } from '#modules/events';
+
+import { createStoryBookmark } from '../../story-bookmark';
+import type { Props as PlaceholderElementProps } from '../components/PlaceholderElement';
+import {
+    type Props as BaseProps,
+    SearchInputPlaceholderElement,
+} from '../components/SearchInputPlaceholderElement';
```

Comment:
I would have imported these from `../components` instead 👀 But I'm not sure what's the rule of thumb here really :) I guess it's more of a personal preference.

Reply:
There is no `index.ts` file in `../components` :)

I think, it was intentionally omitted to avoid re-throwing every component and its `Props` types, when it's possible to just use a deeper import. 

Not perfect, but does the job :man_shrugging: